src/main/java/lotto_auto/model/LottoNumbers.java

```diff
@@ -53,16 +72,19 @@ private void throwIfDuplicate(List<LottoNumber> numbers) {
         }
     }
 
-    private void throwIfNull(List<LottoNumber> numbers) {
-        if (numbers == null) {
-            throw new IllegalArgumentException(ErrorMessage.NOT_NULL_LOTTO_NUMBER);
-        }
+    public List<LottoNumber> export() {
+        return Collections.unmodifiableList(this.lottoNumbers);
     }
 
-    private void throwIfInValidLottoNumberCount(List<LottoNumber> numbers) {
-        if (numbers.size() != LOTTO_NUMBER_COUNT) {
-            throw new IllegalArgumentException(ErrorMessage.INVALID_LOTTO_NUMBER_COUNT);
-        }
+    public boolean contains(LottoNumber lottoNumber) {
+        return this.lottoNumbers.contains(lottoNumber);
+    }
+
+    public int computeMatchCount(LottoNumbers lottoNumbers) {
+        return (int) lottoNumbers.export()
```

Comment:
자기 자신이므로 바로 인스턴스 필드에 접근하면 좋을 것 같아요 :)
```suggestion
        public int computeMatchCount(LottoNumbers winningNumbers) {
            return (int) winningNumbers.lottoNumbers
```

Reply:
헛.. private 지정 을 해둔 상태인데 이게 가능하네요 ..ㅇㅅㅇ;

다른 객체이지만 같은 클래스 이기 떄문에 가능한 경우일까요..?

Reply:
맞습니다. heap 메모리상에서는 다른 객체지만, 같은 클래스 정보를 참조하고 있기 때문에 접근이 가능합니다 ㅎㅎ